src/controllers/user.js

```diff
@@ -1,6 +1,5 @@
 import User from '../domain/user.js'
 import bcrypt from 'bcrypt'
-import { validateTeacherRole } from '../middleware/auth.js'
```

Comment:
explain why this is removed please

Reply:
It was previously used in line 122 inside the updateById function, but its no longer used.

Reply:
Validation is instead done by these helper functions:

```javascript
const studentUpdatingOther = (req) => { ... }
```

```javascript 
const studentUpdatingIllegalFields = (req) => { ... }
```